### DISCUSSION TITLE
Rasberry Pi OS version?

### DISCUSSION THREAD

**Discussion by xanderortega** [rustdesk/rustdesk#9347] [Q&A]
Hello everyone

I'm trying to download the version for my raspberry pi os, but I don't know exactly which one I should download. Could someone help me?

Thanks

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/releases/tag/nightly

![image](https://github.com/user-attachments/assets/33ed5eda-891b-4206-a2e4-6f75ab11111e)

**Reply by xanderortega** [reply]
thanks i appreciate it.
